### PR TITLE
Update production

### PR DESCRIPTION
**Pull request by kinueng**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: src/main/content/_assets/js/javadoc-redirect.js [hunk 1]
```diff
@@ -1,68 +1,94 @@
...
       'https://' +
       window.top.location.hostname +
       port +
       '/docs/latest/reference/javadoc/microprofile-' +
       version +
-      '-javadoc.html#package=' +
+      '-javadoc.html?package=' +
       pack +
       '/package-frame.html&class=' +
       jd;
```
Comment (aknguyen7) [inline src/main/content/_assets/js/javadoc-redirect.js:24]: we can also further simplify the code by putting the content inside if (pack) code in a method ifPackageExists(port, version, pack, jd)

Comment (Sreejith-Websphere) [reply src/main/content/_assets/js/javadoc-redirect.js:24]: I think lets move this now

Comment (Sreejith-Websphere) [reply src/main/content/_assets/js/javadoc-redirect.js:24]: on next PR I will make that refactoring

**Review by Sreejith-Websphere — APPROVED** [review]
lgtm

**Pull request merged**